lib/checkPullRequestReview.js

```diff
@@ -36,9 +36,13 @@ const PTAL_COMMENTS = ['PTAL', 'Please take a look'];
  * @param {import('probot').Octokit.PullsGetReviewResponse} review
  */
 const handleChangesRequested = async (context, pullRequest, review) => {
+  console.log('Inside the function handleChangesRequested');
+  let allLabels = pullRequest.labels.map((label) =>{
```

Comment:
```suggestion
  let allLabels = pullRequest.labels.map((label) => {
```

Reply:
Correcting it in the latest commit Sir.Thanks.